lib/matplotlib/backends/backend_ps.py

```diff
@@ -134,6 +134,70 @@ def _move_path_to_path_or_stream(src, dst):
         shutil.move(src, dst, copy_function=shutil.copyfile)
 
 
+def _font_to_ps_type3(font_path, glyph_ids):
```

Comment:
Even though it's private, can you please give this a docstring that might help a postscript-noob like me getting an idea what this is doing?

Reply:
done

Reply:
Thanks!